core/src/main/java/org/springframework/security/authentication/ott/GenerateOneTimeTokenRequest.java

```diff
@@ -26,15 +26,37 @@
  */
 public class GenerateOneTimeTokenRequest {
 
+	private static final int DEFAULT_EXPIRES_IN = 300;
+
 	private final String username;
 
+	private final int expiresIn;
+
 	public GenerateOneTimeTokenRequest(String username) {
 		Assert.hasText(username, "username cannot be empty");
 		this.username = username;
+		this.expiresIn = DEFAULT_EXPIRES_IN;
```

Comment:
```suggestion
		this(username, DEFAULT_EXPIRES_IN);
```

Delegating to the other constructor ensures we don't have to keep the validation in two places